04-call-summaries/chunking.py

```diff
@@ -0,0 +1,158 @@
+from langchain_text_splitters import (
+    RecursiveCharacterTextSplitter,
+    NLTKTextSplitter,
+    SpacyTextSplitter,
+)
+from langchain_core.prompts import PromptTemplate
+from langchain.docstore.document import Document
+from llm import google_gemini_client, claude_client, gpt_client, ollama_client
+from run import get_transcript
+
+
+# split text into chunks
+def get_text_chunks(text, chunk_size, chunk_overlap, text_splitter_choice):
+    if text_splitter_choice == "2":
+        text_splitter = NLTKTextSplitter()
```

Comment:
Note that the current code always uses `"2"`, so the chunk_size and chunk_overlap are not used.